index.html

```diff
@@ -5,8 +5,75 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="Victor's Portfolio - Web Developer" />
+    <link rel="stylesheet" href="./styles.css" />
     <link rel="shortcut icon" href="./assets/favicon.png" type="image/x-icon" />
     <title>My portafolio</title>
   </head>
-  <body></body>
+  <body>
+    <header class="toolbar">
+      <div class="myLogo">My Logo</div>
+      <div class="nav">
+        <img
+          src="./assets/Enabled.svg"
+          alt="hamburger icon"
+          width="40"
+          height="40"
+        />
+      </div>
+    </header>
+    <section class="headline">
+      <h1>I'm Victor</h1>
+      <h1>Glad to see you</h1>
+      <p class="mainText">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socialMedia">
+        <h2>LET'S CONNECT</h2>
+        <div class="socialMediaImages">
```

Comment:
- [ ] Again, for semantic and search engine optimization purposes, I suggest you change this container to an unordered list (`<ul>`) and its elements to list objects.